matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/aggregation/livelocation/DefaultLiveLocationAggregationProcessor.kt

```diff
@@ -60,22 +60,27 @@ internal class DefaultLiveLocationAggregationProcessor @Inject constructor() : L
         aggregatedSummary.isActive = content.isLive
     }
 
-    override fun handleBeaconLocationData(realm: Realm, event: Event, content: MessageBeaconLocationDataContent, roomId: String, isLocalEcho: Boolean) {
+    override fun handleBeaconLocationData(
```

Comment:
Not sure we should have an interface for this

Reply:
Removed, see [4a23109](https://github.com/vector-im/element-android/pull/5989/commits/4a2310954acf5fa36e429f76c896b1210d48ce0a)